docs/source/algorithms/MaskBTP-v1.rst

```diff
@@ -10,27 +10,27 @@
 Description
 -----------
 
-Algorithm to mask detectors in particular banks, tube, or pixels. It
-applies to the following instruments only: ARCS, CNCS, CORELLI, HYSPEC, NOMAD,
-POWGEN, SEQUOIA, SNAP, TOPAZ. For instruments with rectangular position
-sensitive detectors (POWGEN, SNAP, TOPAZ), the tube is corresponding to
-the x coordinate, and pixel to the y coordinate. For example, on SNAP
-Bank="1", Tube="3" corresponds to 'SNAP/East/Column1/bank1/bank1(x=3)',
-and Bank="1", Tube="3", Pixel="5" is 'SNAP/East/Column1/bank1/bank1(x=3)/bank1(3,5)'.
-
-If one of Bank, Tube, Pixel entries is left blank, it will apply to all
+Algorithm to mask detectors in particular banks, tube, or pixels.
+To use the ``Bank`` argument, the instrument must be one of: ARCS, BIOSANS, CG2 CNCS, CORELLI, EQ-SANS HYSPEC, MANDI, NOMAD, POWGEN, SEQUOIA, SNAP, SXD, TOPAZ, WAND, WISH.
```

Comment:
Instrument scientist (and me :smile: ) prefer using string `EQSANS` instead of `EQ-SANS`.

Reply:
Either will work in the `InstrumentName` parameter.